src/ws/cockpitwebservice.c

```diff
@@ -1017,7 +1017,7 @@ on_session_recv (CockpitTransport *transport,
   if (session == NULL)
     {
       /* This is not an error since channel closing can race */
-      g_debug ("dropping message with unknown channel %s from session", channel);
+      g_warning ("dropping message with unknown channel %s from session", channel);
```

Comment:
What is the reason for changing this? Also the comment no longer matches the code.